README.md

```diff
@@ -21,15 +21,27 @@ Regex for pages like someuser.github.io.
 Use https://api.github.com/users/{user_login} (60 requests/hour unauthenticated)
 
 ## Linkedin
-    http(s)?:\/\/([\w]+\.)?linkedin\.com\/in\/[A-z0-9_-]+\/?
+
+    http(s)?:\/\/([\w]+\.)?linkedin\.com\/in\/[\w\-À-ÿ%]+(\/.*)?
 RegEx for public URLs.
 
-    http(s)?:\/\/([\w]+\.)?linkedin\.com\/pub\/[A-z0-9_-]+(\/[A-z0-9]+){3}\/?
+    http(s)?:\/\/([\w]+\.)?linkedin\.com\/pub\/[\w\-À-ÿ%]+(\/\w+){3}(\/.*)?
```

Comment:
Hey @Ronserruya, thanks for the PR. Could you help me understand the `À-ÿ` part?

Reply:
Sure, \w gets all English letters, but you need À-ÿ to catch accented letters, that's for people with these letters in their name, for example 

www.linkedin.com/in/mark-simon-müller-81a83743